# Value of the parameter in ActionParameter

# The type of this parameter
#
# Field Type: Required
uint8 type

# Enum values for type
#
# Field Type: Required
uint8 TYPE_ARRAY=1
uint8 TYPE_BOOL=2
uint8 TYPE_NUMBER=3
uint8 TYPE_STRING=4
uint8 TYPE_OBJECT=5

# Value that this parameter holds
#
# Field Type: Required
string value

